machine/os/ubuntu.go

```diff
@@ -0,0 +1,37 @@
+package os
+
+import (
+	"strings"
+
+	"github.com/docker/machine/drivers"
+)
+
+func init() {
+	RegisterRuntime("ubuntu", &RegisteredRuntime{
+		Detect: UbuntuDetection,
+	})
+}
+
+func UbuntuDetection(d drivers.Driver) (*Runtime, error) {
+	out, err := sshCommand(d, "lsb_release -i | cut -f2 -d:")
```

Comment:
Why not just cat the whole file and parse it in Go so you can use other fields later if you need to?